src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -288,6 +289,9 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
         experimentVariant,
       ),
   );
+
+  const readTime = pageData?.metadata?.stats?.readTime || undefined;
```

Comment:
don't we need to adjust this per language according to data [like this](https://www.readingtimeestimator.com)?